src/main/java/fi/aalto/cs/apluscourses/ui/repl/ReplConfigurationForm.java

```diff
@@ -43,7 +43,8 @@ public ReplConfigurationForm() {
    */
   public ReplConfigurationForm(@NotNull ReplConfigurationFormModel model) {
     this.model = model;
-    dontShowThisWindowCheckBox.setSelected(PluginSettings.isShowReplConfigurationDialog());
+    dontShowThisWindowCheckBox.setSelected(
+        PluginSettings.getInstance().shouldShowReplConfigurationDialog());
```

Comment:
why the change? :confused:

Reply:
The (renamed) method was initially declared static though it shouldn't have been. `PluginSettings` is a singleton class.